lib/p2p/Pool.ts

```diff
@@ -47,8 +50,10 @@ class Pool extends EventEmitter {
   /** This node's listening external socket addresses to advertise to peers. */
   private addresses?: Address[];
 
-  constructor(config: PoolConfig, private logger: Logger, db: DB) {
+  constructor(config: PoolConfig, private logger: Logger, db: DB,
+              public lndBtcClient:LndClient, public lndLtcClient:LndClient) {
```

Comment:
Maybe make these client fields optional with the `?` operator so that the tests will compile, eventually I'd imagine this will be refactored such that lnd clients are not required parameters to create a p2p pool anyway.